app/models/action_mailbox/inbound_email/message_id.rb

```diff
@@ -27,7 +27,7 @@ def create_and_extract_message_id!(source, **options)
       def extract_message_id(source)
         Mail.from_source(source).message_id
       rescue => e
-        # FIXME: Add logging with "Couldn't extract Message ID, so will generating a new random ID instead"
+        logger.warn "Couldn't extract Message ID, so will generating a new random ID instead"
```

Comment:
Does logger.warn return nil? Needs to in order for the conditional to work.

Reply:
Ahh you are right, thanks. I've added return nil explicitly, not sure do we need to cover this case in the tests, wdyt?
We do have a [test](https://github.com/rails/actionmailbox/blob/master/test/unit/inbound_email/message_id_test.rb#L11) but it covers `Mail.from_source(source).message_id #==> nil`, not the exception case

Reply:
Does the `e` exception contain something useful here? Should we include `message` and `backtrace` in the log? Otherwise we should just remove that needless variable.

`so will generating` isn't grammatically right, how about `"Couldn't extract Message ID from email, generating new one instead."` for the message as a whole?